### PR TITLE
U256, H256, Address conversions added

### PR DESCRIPTION
**Pull request by fckt**

### CODE REVIEW

**Comment by NikVolf** [comment]
don't import pwasm-std, import parity-hash/bigint please

File: tests/Cargo.toml [hunk 1]
```diff
@@ -6,3 +6,4 @@ authors = ["NikVolf <nikvolf@gmail.com>"]
 [dependencies]
 pwasm-abi = { path = ".." }
 pwasm-abi-derive = { path = "../derive" }
+pwasm-std = { git = "https://github.com/nikvolf/pwasm-std", features = ["std"] }
```
Comment (NikVolf) [inline tests/Cargo.toml]: what's this for?

File: tests/src/lib.rs [hunk 2]
```diff
@@ -16,10 +16,11 @@ use std::borrow::Cow;
 
 extern crate pwasm_abi;
 extern crate pwasm_abi_derive;
+extern crate pwasm_std;
 
 use pwasm_abi_derive::eth_dispatch;
 
-type U256 = [u8; 32];
+use pwasm_std::bigint::U256;
```
Comment (NikVolf) [inline tests/src/lib.rs]: ?

File: tests/Cargo.toml (hunk 1 above)
Comment (fckt) [reply tests/Cargo.toml]: just a mistake, nwm

**Comment by NikVolf** [comment]
cool

**Pull request merged**